From 5a0dfe5f42f6c7b6fc07e2aca9dce2bad61e624e Mon Sep 17 00:00:00 2001
From: "David Kaspar [Dee'Kej]" <dkaspar@redhat.com>
Date: Fri, 7 Oct 2016 12:57:06 +0200
Subject: [PATCH 2/6] Make sure customizations to
 /etc/ghostscript/Fontmap.local are used

  For more info, see: https://bugzilla.redhat.com/show_bug.cgi?id=610301
---
 Resource/Init/Fontmap | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Resource/Init/Fontmap b/Resource/Init/Fontmap
index ae5af96e0..72aeb1fa3 100644
--- a/Resource/Init/Fontmap
+++ b/Resource/Init/Fontmap
@@ -2,3 +2,5 @@
 % See Fontmap.GS for the syntax of real Fontmap files.
 %% Replace 1 (Fontmap.GS)
 (Fontmap.GS) .runlibfile
+% must be at the bottom of line to allow people overriding everything.
+(Fontmap.local) .runlibfileifexists
-- 
2.31.0.208.g409f899ff0-goog

